platform_plugin_aspects/management/commands/monitor_load_test_tracking.py

```diff
@@ -0,0 +1,341 @@
+        Get the current state of redis for this iteration.
+        """
+        r = redis.Redis.from_url(settings.EVENT_BUS_REDIS_CONNECTION_URL)
+        info = r.xinfo_stream("openedx-analytics", full=True)
+
+        lag = []
+        for g in info["groups"]:
+            lag.append({str(g["name"]): g["lag"]})
+
+        consumer_stats = {
+            "total_events": info["length"],
+            "queue_lengths": lag,
+        }
+
+        log.info(f"Redis bus queue length: {consumer_stats['queue_lengths']}")
+
+        return consumer_stats
+
+    def get_kafka_bus_stats(self):
+        """
+        Get the current state of ClickHouse for this iteration.
+        """
+        if not confluent_kafka:  # pragma: no cover
+            raise CommandError(
+                "Trying to monitor Kafka bus, but confluent_kafka is not installed"
+            )
+
+        brokers = settings.EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS
+        topic = f"{settings.EVENT_BUS_TOPIC_PREFIX}-analytics"
+        group = "analytics-service"
+
+        # This consumer will not join the group, but the group.id is required by
+        # committed() to know which group to get offsets for.
+        consumer = confluent_kafka.Consumer(
+            {"bootstrap.servers": brokers, "group.id": group}
+        )
+
+        # Get the topic's partitions
+        metadata = consumer.list_topics(topic, timeout=10)
+
+        if metadata.topics[topic].error is not None:  # pragma: no cover
+            log.info(metadata.topics[topic].error)
+
+        partitions = [
+            confluent_kafka.TopicPartition(topic, p)
+            for p in metadata.topics[topic].partitions
+        ]
+        committed = consumer.committed(partitions, timeout=10)
+
+        consumer_stats = {
+            "topic": topic,
+            "partitions": [],
+        }
+
+        for partition in committed:
+            # Get the partitions low and high watermark offsets.
+            low, high = consumer.get_watermark_offsets(
+                partition, timeout=10, cached=False
+            )
+
+            if high < 0:  # pragma: no cover
+                lag = 0
+            elif partition.offset < 0:  # pragma: no cover
+                # No committed offset, show total message count as lag.
+                # The actual message count may be lower due to compaction
+                # and record deletions.
+                lag = high - low
+            else:
+                lag = high - partition.offset
+
+            log.info(f"{partition.topic} [{partition.partition}] Lag: {lag}")
+
+            consumer_stats["partitions"].append(
+                {
+                    "partition": partition.partition,
+                    "lag": lag,
+                }
+            )
+
+        consumer.close()
+        return consumer_stats
+
+    def _call_vector_graphql(self):
+        """
+        Make the actual GraphQL call to the Vector API.
+        """
+        # FIXME: Pull this from settings
```

Comment:
I think this is the last thing I need to update, but it involves plumbing the Tutor settings all the way through to here.